### PR TITLE
[Encore] Webpack Dev Server: live reload & HMR

### PR DESCRIPTION
**Pull request by ne0-cz**
To use both live reload and HMR, some options are required and I think the docs would be a perfect place to find them.

### CODE REVIEW

**Comment by javiereguiluz** [comment]
Thanks for this proposal.

Before merging, let's ping some JavaScript expert (like @stof or @weaverryan) to see if they can review these changes. Thanks!

**Comment by weaverryan** [comment]
I've never considered having the browser "live reload" when you change PHP or Twig files. Is that a common thing people want/do?

**Comment by ne0-cz** [comment]
I don't really know what people want/do, I can only speak about my experience - a full-stack developer.

When I work on some JS-related feature I'll be definitely editing both the JS files and Twig templates. And since the JS changes are automatically updated in the browser without needing to manually reload it, I'm used to the same behavior even with template changes.

The `static.watch` option wasn't easy to figure out. This is exactly the place that'd help me solve it..

File: frontend/encore/dev-server.rst [hunk 1]
```diff
@@ -128,6 +128,34 @@ your page. HMR works automatically with CSS (as long as you're using the
 ``dev-server`` and Encore 1.0 or higher) but only works with some JavaScript
 (like :doc:`Vue.js </frontend/encore/vuejs>`).
 
+Enabling live reload with HMR
```
Comment (weaverryan) [inline frontend/encore/dev-server.rst]:
```suggestion
Live Reloading when changing PHP / Twig Files
```

**Review by weaverryan — COMMENTED** [review]
I'd propose changing the section title. But otherwise, sure, this might be useful 👍

**Review by weaverryan — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Nice doc update! Luděk thanks and congrats on your first Symfony Docs contribution 🎉  Thanks also to Ryan for his review!

**Pull request merged**